journal/modernized/37-chapter-34.adoc

```diff
@@ -671,6 +671,9 @@ will lastingly remain;--that He may be glorified thereby.
 [.offset]
 The following are extracts from the address to the missionaries in New Zealand.
 
+[.embedded-content-document.address]
```

Comment:
nice job flagging this one as an `address` and the last one as `epistle`, perfect. 👍😊